logic/src/commonMain/kotlin/com/wire/kalium/logic/data/message/MessageMapper.kt

```diff
@@ -326,8 +324,23 @@ class MessageMapperImpl(
         // We don't care about the content of these messages as they are only used to perform other actions, i.e. update the content of a
         // previously stored message, delete the content of a previously stored message, etc... Therefore, we map their content to Unknown
         is MessageContent.Knock -> MessageEntityContent.Knock(hotKnock = regularMessage.hotKnock)
+        is MessageContent.Composite -> MessageEntityContent.Composite(
+            text = regularMessage.textContent?.let(this::toTextEntity),
```

Comment:
nice one, my personal thing but I am allergic to this :P it is esthetic thing but I prefer to not use this when not necessary looks more pleasing to eyes